en/docs/try-out/account-information-service-flow.md

```diff
@@ -0,0 +1,896 @@
+-H 'Authorization: Bearer <APPLICATION_ACCESS_TOKEN>' \
+-H 'Content-Type: application/json; charset=UTF-8' \
+-H 'Accept: */*' \
+--cert <TRANSPORT_PUBLIC_KEY_FILE_PATH> --key <TRANSPORT_PRIVATE_KEY_FILE_PATH> \
+-d '{
+    "access": {
+        "accounts": [
+            {
+                "iban": "DE98765432109876543210"
+            }
+        ],
+        "balances": [
+            {
+                "iban": "DE98765432109876543210"
+            }
+        ],
+        "transactions": [
+            {
+                "iban": "DE98765432109876543210"
+            }
+        ]
+    },
+    "recurringIndicator": true,
+    "validUntil": "2022-03-22",
+    "frequencyPerDay": 4,
+    "combinedServiceIndicator": false
+}'
+```
+
+2. The response contains a Consent Id. A sample response is as follows:
+
+    ```
+    {
+       "consentId":"fe3efd03-93c9-4b6b-8f9c-e1d16ce054da",
+       "chosenScaMethod":{
+          "authenticationVersion":"1.0",
+          "name":"SMS OTP on Mobile",
+          "authenticationType":"SMS_OTP",
+          "explanation":"SMS based one time password",
+          "authenticationMethodId":"sms-otp"
+       },
+       "_links":{
+          "scaStatus":{
+             "href":"/v1/consents/fe3efd03-93c9-4b6b-8f9c-e1d16ce054da/authorisations/0dfb97c4-53c3-4134-a9ca-5ad1ba41edb9"
+          },
+          "scaOAuth":{
+             "href":"https://localhost:8243/.well-known/openid-configuration"
+          },
+          "self":{
+             "href":"/v1/consents/fe3efd03-93c9-4b6b-8f9c-e1d16ce054da"
+          },
+          "status":{
+             "href":"/v1/consents/fe3efd03-93c9-4b6b-8f9c-e1d16ce054da/status"
+          }
+       },
+       "consentStatus":"received"
+    }
+    ```
+
+#### Account references
+
+- An account reference is used to address specific accounts. It is a combination of an account identifier and optionally 
+a currency. 
+- Account references are only used in the context of **Dedicated accounts consents** to define the specific accounts 
+that the consent requires.
+- WSO2 Open Banking Berlin toolkit supports IBAN, BBAN and Masked PAN account identifier types. These can be configured 
```

Comment:
The toolkit can also support the PAN as an account identifier. But this needs to be handled using a customization when integrating with the banking backend. This information will be included in the "Core Banking Integration" documentation (currently WIP). Hope we can update this doc after the core banking integration doc is published.

Reply:
hi @Akila94, noted with thanks. I created this issue to track this doc requirement -https://github.com/wso2/docs-open-banking/issues/567.